### PR TITLE
docs: update DSP refs

### PR DESCRIPTION
**Pull request by rajamatt**
Closes https://github.com/unoplatform/private/issues/443

## Description (*)

1) Updates DSP references to Uno DSP.
2) Updates usage of zip files to JSON theme files.

## Contribution checklist (*)
 - [ ] I have read the Contributing Guidelines
 - [ ] Commits have been made with meaningful commit messages
 - [ ] All automated tests have passed successfully

### CODE REVIEW

File: tube-player/modules/06-Theme-overrides/README.md [hunk 1]
```diff
@@ -59,15 +59,15 @@ Uno Platform offers two ways to import app themes.
 
 ### Create a custom theme and import DSP
 
-DSP stands for [Design System Package](https://github.com/AdobeXD/design-system-package-dsp) and is a conventional way to transport design themes between different systems.
-There are several theme builders out there, the [Material Theme Builder](https://m3.material.io/theme-builder#/custom) is one of them, once a theme is created, it can be exported as a DSP file (it's a compressed *.zip* archive that includes all the data needed).
+DSP stands for Design System Package and is a conventional way to transport design themes between different systems.
```
Comment (carldebilly) [inline tube-player/modules/06-Theme-overrides/README.md]: !!

**Review by carldebilly — APPROVED** [review]

File: tube-player/README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ This app was developed using Uno Platform and draws inspiration from [Naweed Akr
 - How to create a new solution using the Uno Platform dotnet new template or the Uno Platform Template Wizard.
 - How to use the [Uno Platform Figma plugin](https://aka.platform.uno/uno-figma) to generate your UI in C# Markup from [this Figma file](https://aka.platform.uno/uno-figma-tubeplayer-workshop) (optional).
 - How to build your app's presentation layer using C# Markup and Model-View-Update-eXtended (MVUX).
-- How to customize your app theme and override its theme colors by using either C# code imported from Figma, or by importing a custom DSP ([Design System Package](https://github.com/AdobeXD/design-system-package-dsp)) color theme.
+- How to customize your app theme and override its theme colors by using either C# code imported from Figma, or by importing a custom Uno DSP (Design System Package) color theme.
```
Comment (agneszitte) [inline tube-player/README.md]:
```suggestion
- How to customize your app theme and override its theme colors by using either C# code imported from Figma, or by importing a custom Uno DSP color theme.
```

File: tube-player/modules/06-Theme-overrides/README.md [hunk 3]
```diff
@@ -40,12 +40,12 @@ Uno Platform offers two ways to import app themes.
 
 ### Import C#
 
-1. Head over to the IDE and delete the *ColorPaletteOverride.zip* file. This won't be needed as you're importing the color overrides from Figma. It's used when importing a DSP theme from an external editor (switch to the <!--TODO make sure link works -->[DSP tab](#tab/export/dsp) for more).
+1. Head over to the IDE and delete the *ColorPaletteOverride.json* file. This won't be needed as you're importing the color overrides from Figma. It's used when importing a Uno DSP theme from an external editor (switch to the <!--TODO make sure link works -->[DSP tab](#tab/export/dsp) for more).
```
Comment (agneszitte) [inline tube-player/modules/06-Theme-overrides/README.md]:
```suggestion
1. Head over to the IDE and delete the *ColorPaletteOverride.json* file. This won't be needed as you're importing the color overrides from Figma. It's used when importing an Uno DSP theme from an external editor (switch to the [DSP tab](#tab/export/dsp) for more information).
```

File: tube-player/modules/06-Theme-overrides/README.md [hunk 4]
```diff
@@ -55,19 +55,19 @@ Uno Platform offers two ways to import app themes.
...
 
-# [DSP](#tab/export-dsp)
+# [Uno DSP](#tab/export-dsp)
 
-### Create a custom theme and import DSP
+### Create a custom theme and import Uno DSP
 
-DSP stands for [Design System Package](https://github.com/AdobeXD/design-system-package-dsp) and is a conventional way to transport design themes between different systems.
-There are several theme builders out there, the [Material Theme Builder](https://m3.material.io/theme-builder#/custom) is one of them, once a theme is created, it can be exported as a DSP file (it's a compressed *.zip* archive that includes all the data needed).
+Uno DSP (Design System Package) is a conventional way to transport design themes between different systems.
```
Comment (agneszitte) [inline tube-player/modules/06-Theme-overrides/README.md]:
```suggestion
Uno DSP is a conventional way to transport design themes between different systems.
```

File: tube-player/modules/06-Theme-overrides/README.md (hunk 3 above)
Comment (agneszitte) [reply tube-player/modules/06-Theme-overrides/README.md]: @rajamatt you can run the doc locally to make sure the link is working ;)

**Review by carldebilly — APPROVED** [review]

**Review by agneszitte — APPROVED** [review]

**Pull request merged**